.github/workflows/ci.yml

```diff
@@ -18,22 +18,13 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        toolchain: ["1.41.0", "1.46.0", stable, beta]
+        toolchain: ["1.46.0", stable, beta]
         profile: ['', --release]
         features: ['', '--all-features']
         exclude:
           # capture-stream is not supported on Windows or compiler versions too old for Tokio.
           - os: windows-latest
             features: '--all-features'
-          - os: windows-latest
-            toolchain: "1.41.0"
-          # 1.46.0 is only a special requirement on Windows
-          - os: ubuntu-latest
-            toolchain: "1.46.0"
-          - os: macos-latest
-            toolchain: "1.46.0"
-          - toolchain: "1.41.0"
```

Comment:
You should not remove the exclude of '--all-features' for MSRV as we don't support that. Only the base version is supported on MSRV. The other removals are fine.